backend/tournesol/tests/utils/mock_now.py

```diff
@@ -0,0 +1,30 @@
+from contextlib import contextmanager
```

Comment:
should it be in a more general directory? @GresilleSiffle what do you think?
Should we have a test for this function?

Reply:
> should it be in a more general directory? @GresilleSiffle what do you think?

Hard to say. It looks good to me here, in the `tests` folder of the main Django app, the same way it will look good in the `tests` folder of the core app.

If we want the `tests` folders to only contain tests we could choose to move this class to `tournesol.lib.test` (singular, to not be confused with a folder containing actual tests). This file is very small so I don't mind thinking about the architecture after the merge.

> Should we have a test for this function?

I'm always in favor of adding unit tests in the back end : ) It's ok for me if we add tests after the merge. We can do it together if you want @aidanjungo